common/src/chain/transaction/output/mod.rs

```diff
@@ -100,6 +104,16 @@ pub enum TxOutput {
     CreateDelegationId(Destination, PoolId),
     #[codec(index = 6)]
     DelegateStaking(Amount, DelegationId),
+    #[codec(index = 7)]
+    TokensOp(TokenOutput),
```

Comment:
I introduced a nesting level here but now I'm thinking whether it was a good idea. The motivation was to group them, but this actually is a drawback on its own - you can wildcard this arm which basically wildcards the whole group. Also it's now a bit confusing to has TxOutput::TokensOp, what it even means...
This enum is fundamental so I don't want to change it again, any thoughts?

Reply:
Using wildcards should be prohibited anyway...

Reply:
yes and with this nesting it's easier to ignore that rule

Reply:
I wouldn't worry about that. We should just not wildcard anything. That's why I nitpick on that all the time.